scripts/apps/authoring/media/MediaMetadataEditorDirective.ts

```diff
@@ -76,6 +119,8 @@ export default function MediaMetadataEditorDirective(metadata, deployConfig, fea
              * @return {Boolean}
              */
             scope.isDisabled = (field) => scope.disabled || (scope.associated && field.external);
+
+            scope.isExtra = (field): boolean => field.cv && field.cv.field_type === 'text';
```

Comment:
Isn't a field considered to be "extra" if it can be dynamically configured via UI? I thought field type shouldn't matter. Or should it?

Reply:
it does, because if it's controlled vocabulary it is stored as subject

Reply:
What I'm asking is why do you need `field.cv.field_type === 'text'`? As I understand the only difference between extra fields is that they can be configured dynamically and are stored in `IArticle.extra` namespace. Can't there be other types of extra fields than text?

Reply:
there is also date and urls, have to fix the check

Reply:
Is the check needed in the first place if we check against available types?

Reply:
I think it might be different for type `media` and `related_content`, but will double check